src/Symfony/Bridge/PhpUnit/CoverageListener.php

```diff
@@ -29,7 +29,7 @@ class CoverageListener implements TestListener
     public function __construct(callable $sutFqcnResolver = null, bool $warningOnSutNotFound = false)
     {
         $this->sutFqcnResolver = $sutFqcnResolver ?? static function (Test $test): ?string {
-            $class = \get_class($test);
+            $class = $test::class;
```

Comment:
these changes in the PHPUnit bridge need to be reverted as it still supports PHP 7

Reply:
ack, reverted.

then, I strongly encourage to disable the rule, as one cannot apply current Fixer config in such case and trust to use the results. 
https://github.com/symfony/symfony/pull/52117/commits/aadeffb2b11ca03b8868ae983b8531c8c6a29996

Reply:
Very open to see such PR.
Yet, for now, if John Doe comes, see the Fixer config, apply it and see lot of changes needed, and those changes breaking CI, it's very confusing.